editor/editor_export.cpp

```diff
@@ -252,6 +253,83 @@ String EditorExportPreset::get_script_encryption_key() const {
 
 ///////////////////////////////////
 
+bool EditorExportPlatform::fill_log_messages(RichTextLabel *p_log, Error p_err) {
+	bool has_messages = false;
+
+	int msg_count = get_message_count();
+
+	p_log->add_text(TTR("Project export for the platform") + " \"");
+	p_log->add_image(get_logo(), 16 * EDSCALE, 16 * EDSCALE, Color(1.0, 1.0, 1.0), INLINE_ALIGNMENT_CENTER);
+	p_log->add_text(" ");
+	p_log->add_text(get_name());
+	p_log->add_text("\" - ");
```

Comment:
It's a bit tricky for translation with the platform name added later as some languages might need to place it somewhere else to respect the natural flow of e.g. "Project export for platform Windows". It could use `vformat` but then you can't easily add the logo, which was a nice touch.

Maybe adding a colon to force a syntactic break in the sentence would work better, and the quotes can then be removed:

```suggestion
	p_log->add_text(TTR("Project export for platform:") + " ");
	p_log->add_image(get_logo(), 16 * EDSCALE, 16 * EDSCALE, Color(1.0, 1.0, 1.0), INLINE_ALIGNMENT_CENTER);
	p_log->add_text(" ");
	p_log->add_text(get_name());
	p_log->add_text(" - ");
```